src/client/Microsoft.Identity.Client/AppConfig/TraceTelemetryConfig.cs

```diff
@@ -46,7 +46,7 @@ public TraceTelemetryConfig()
         /// <remarks>This API is experimental and it may change in future versions of the library without an major version increment</remarks>
         public Action<ITelemetryEventPayload> DispatchAction => payload =>
         {
-            var j = new JObject();
+            var j = new JsonObject();
```

Comment:
It's already obsoleted. Can't delete it since it affects public API. https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/b9faf12281d5fb0ff6767128694974f8868d3bc0/src/client/Microsoft.Identity.Client/AppConfig/TraceTelemetryConfig.cs#L18-L20